test/src-tests/index-test.js

```diff
@@ -120,6 +120,10 @@ describe('find-parent/index', function() {
   });
 
   describe('#byClassName', function() {
+    beforeEach(function() {
```

Comment:
They are using it. They're currently benefitting from the fact element is defined on preceding beforeEach blocks.